lib/browser/remote/server.ts

```diff
@@ -373,12 +373,12 @@ const logStack = function (contents: electron.WebContents, code: string, stack:
 }
 
 handleRemoteCommand('ELECTRON_BROWSER_WRONG_CONTEXT_ERROR', function (event, contextId, passedContextId, id) {
-  const objectId = [passedContextId, id]
+  const objectId: [string, number] = [passedContextId, id]
   if (!rendererFunctions.has(objectId)) {
     // Do nothing if the error has already been reported before.
     return
   }
-  removeRemoteListenersAndLogWarning(event.sender, rendererFunctions.get(objectId))
+  removeRemoteListenersAndLogWarning(event.sender, rendererFunctions.get(objectId) as (...args: any[]) => void)
```

Comment:
We should be able to get rid of this cast by declaring the correct type on L26, e.g.:

```ts
const rendererFunctions = v8Util.createDoubleIDWeakMap<Function>()
```